.github/actions/build-jtreg/action.yml

```diff
@@ -52,8 +52,24 @@ runs:
 
     - name: 'Build JTReg'
       run: |
-        # Build JTReg and move files to the proper locations
-        bash make/build.sh --jdk "$JAVA_HOME_17_X64"
+        # Try building JTReg several times, backing off exponentially on failure.
+        # ~500 seconds in total should be enough to capture most of the transient
```

Comment:
Since building jtreg will block building the code (not just running tests), this feels like a potentially quite long waiting time. 

Are we sure waiting even helps? What kind of issues is it we are encoutering? Downloading resources from the net?

Reply:
Well, we are seeing transient network errors like:

```
[build.sh][INFO] Downloading https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.8-bin.zip to /home/runner/work/jdk/jdk/jtreg/src/make/../build/deps/apache-ant-1.10.8-bin.zip
Error: sh][ERROR] wget exited with exit code 4 
```

wget exit code 4 means "Network failure". We "know" waiting helps in the sense that job restarts help.